Dockerfile

```diff
@@ -1,36 +1,20 @@
-# take default image of node boron i.e  node 6.x
-FROM node:8.9 as builder
-RUN npm i -g yarn
-
-# create app directory in container
-RUN mkdir -p /app
+# node image
+FROM node:8.10.0
 
 # set /app directory as default working directory
-WORKDIR /app
-
-# only copy package.json initially so that `RUN yarn` layer is recreated only
-# if there are changes in package.json
+WORKDIR /app/
 COPY . /app/
 
-RUN yarn
-
-# compile to ES5
-RUN yarn build
-
-FROM node:8.9 
-
-WORKDIR /app
-
-COPY --from=builder /app /app
-
 # set up public and private keys
 RUN echo -e 'y\n'|ssh-keygen -q -t rsa -b 4096 -N "" -f private.key &&\
     openssl rsa -in private.key -pubout -outform PEM -out private.key.pub
 
-#USER 50000:50000
+RUN yarn install --pure-lockfile && \
```

Comment:
Reduces image size by containing to one slice. :+1:

Reply:
As in, have it all one line?